zilliqa/src/consensus.rs

```diff
@@ -37,9 +43,8 @@ use crate::{
 
 #[derive(Debug)]
 struct NewViewVote {
-    signatures: Vec<NodeSignature>,
-    signers: Vec<u16>,
-    cosigned: BitVec,
+    // Represents signer index as kept in deposit contract and received signature
+    signatures: BTreeMap<u16, NodeSignature>,
```

Comment:
I still don't really understand why this is a map and not just a `BitVec` of signers, given that we need to convert it to a bitvec later anyway and that the bitvec is probably more efficient because we expect a high proportion of the committee to be signers.